hibernate-core/src/main/resources/org/hibernate/hibernate-mapping-4.0.xsd

```diff
@@ -65,11 +65,13 @@ arbitrary number of queries, and import declarations of arbitrary classes.
                 <xs:complexType>
                   <xs:attribute name="name" use="required" type="xs:string"/> <!-- The parameter name -->
                   <xs:attribute name="type" use="required" type="xs:string"/> <!-- The parameter type -->
+                  <xs:attribute name="resolver" type="xs:string"/> <!-- The parameter resolver -->
```

Comment:
Please revert the changes to this file. hbm.xml is deprecated and will most likely be removed in 7.0.